setup.cfg

```diff
@@ -1,5 +1,57 @@
-[bdist_wheel]
-universal = 1
-
 [metadata]
+name = plumbum
+author = Tomer Filiba
+author_email = tomerfiliba@gmail.com
+license = MIT
+description = Plumbum: shell combinators library
+keywords =
+    path,
+    local,
+    remote,
+    ssh,
+    shell,
+    pipe,
+    popen,
+    process,
+    execution,
+    color,
+    cli
+url = https://plumbum.readthedocs.io
+long_description = file: README.rst
+long_description_content_type = text/x-rst
 license_file = LICENSE
+classifiers =
+    Development Status :: 5 - Production/Stable
+    License :: OSI Approved :: MIT License
+    Operating System :: Microsoft :: Windows
+    Operating System :: POSIX
+    Programming Language :: Python :: 2.6
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.4
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Topic :: Software Development :: Build Tools
+    Topic :: System :: Systems Administration
+platforms = POSIX, Windows
+provides = plumbum
+
+[options]
+packages =
+    plumbum
+    plumbum.cli
+    plumbum.commands
+    plumbum.machines
+    plumbum.path
+    plumbum.fs
+    plumbum.colorlib
+setup_requires = setuptools>=44; wheel; setuptools_scm[toml]>=3.4.3
```

Comment:
Please remove this line. `setup_requires` is deprecated.

Reply:
I don't think we should remove it (older versions of setuptools that don't know about pyproject.toml should understand that).

But as you wish, it's not a very strong issue, ones staying on bleeding edge (like me) should not encounter any issues from that.

Reply:
It turned out without this line build fails.

Reply:
Okay. Should only affect building wheels, which only devs and CI does, but maybe we don’t have the CI up to date. It doesn’t hurt to have it.